packages/components/src/caption/index.native.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
If I understand correctly, this file seems to be a duplicate, right?
Since we are using the component from the `block-editor` package, this one seems to not be necessary 🤔

Reply:
Would've sworn I removed that. Thanks for catching it.